cli/install.go

```diff
@@ -205,8 +205,62 @@ type githubTaggedReleaseResponse struct {
 }
 
 type gitHubAsset struct {
-	Url  string // download url for asset
-	Name string // contains platform information about binary
+	Url                  string // url for asset json object
+	Name                 string // contains platform information about binary
+	Browser_download_url string // download url
+}
+
+func GetZlsDownloadUrl(version string, archDouble string) (string, error) {
```

Comment:
Does this need to be public? All functions that are only used in a single module should be private to that module.

`func getZLSDownloadURL`

Reply:
Didn't know go worked like that 😅